Gopkg.lock

```diff
@@ -604,10 +604,11 @@
   revision = "8dd112bcdc25174059e45e07517d9fc663123347"
 
 [[projects]]
-  digest = "1:b20c63a56900e442d5f435613fefc9392cbe8849467510fcc3869dbdad9441bb"
+  digest = "1:514d2147b3fd915870b534e3578feed1c40a82aa751a015b160785616bfa2130"
```

Comment:
@jcvenegas -- just because i'm lame -- What/why did we need to update? Were we using openapi before?

Reply:
It was missing `golang.org/x/oauth2` added to the commit message